index.html

```diff
@@ -24,6 +25,7 @@
                 <li><a class="nav-menu" href="#contact-section">Contact</a></li>
             </ul>
             <button type="button" id="hamburger"><img src="https://raw.githubusercontent.com/mwismann/mwismann.github.io/a5668d2e5a89784262bf1747248c72134396cb1c/icon-img/hamburger.svg" alt="Options Menu"></button>
+            <button type="button" id="close-mobile-menu"><img src="icon-img/icon-cancel.svg" alt="Close Mobile Menu"></button>
```

Comment:
- [Optional] Dear @mwismann, Great job making the mobile menu, but as you can see, your menu behaves differently on the desktop version. Please remember we need to have the same design for both the desktop and mobile versions. 🙌
![Screenshot 2023-01-31 020044](https://user-images.githubusercontent.com/69077061/215616213-86fbeea2-e4f2-48db-b1f8-4ff38f64a08c.png)